tests/cypress/e2e/features/skeletons_pipeline.js

```diff
@@ -223,5 +223,18 @@ context('Manipulations with skeletons', { scrollBehavior: false }, () => {
 
             cy.removeAnnotations();
         });
+
+        it('Copy/paste a skeleton shape', () => {
+            cy.on('window:console:error', (error) => {
+                throw new Error(`Console error detected: ${error}`);
+            });
+
+            createSkeletonObject('shape');
+            cy.interactAnnotationObjectMenu('#cvat-objects-sidebar-state-item-1', 'Make a copy');
+            cy.get('.cvat-canvas-container').click();
+            cy.get('#cvat_canvas_shape_5').should('exist').and('be.visible');
```

Comment:
The scenario looks a bit wrong. Even before the fix it runs smoothly. We need to activate the exact point on the skeleton, and run a copy action, not just a skeleton.
Consider:
![skeleton-crush](https://github.com/cvat-ai/cvat/assets/50956430/514da985-9878-41bd-a29c-fea26e4a411f)

Reply:
Ou, I misunderstood the fixed bug. Remade the test.